lively.collab/tests/browser-test.js

```diff
@@ -109,6 +110,43 @@ describe('comment browser', function () {
   });
 });
 
+describe('comment indicator', function () {
+  let morph, browser, comment;
+  const exampleText = 'Example text';
+  const exampleName = 'a test morph';
+
+  beforeEach(async function () {
+    morph = new Morph();
+    morph.name = exampleName;
+    new CommentBrowser();
+    browser = CommentBrowser.instance; // This shouldn't be neccessary
+    await CommentBrowser.whenRendered();
+    comment = await morph.addComment(exampleText);
+  });
+
+  it('is visible when browser is open', function () {
+    CommentBrowser.open();
+    expect($world.submorphs.filter((submorph) => submorph.isCommentIndicator).length == 1).to.be.ok;
```

Comment:
Could be made stronger when checking the difference after a new comment is created